docs/src/introduction/getting-started.md

```diff
@@ -45,80 +51,131 @@ Before starting, ensure you have the following:
+
+    ```shell
+    miden-client tx new consume-notes <Account-Id> <Note-Id>
+    ```
+
+    You should be able to find your account and note id by listing both `accounts` and `input-notes`:
+
+    ```shell
+    miden-client account -l
+    miden-client input-notes -l
+    ```
+    
+2. **Visualising account vault:** After successfully running the previous commands your account should now contained the tokens sent from the faucet. You can visualise your accounts vault by running the following command: 
+
+    ```shell
+    miden-client account show <Account-Id> -v
+    ```
+    You should now see your accounts vault containing the funds sent by the faucet. 
+
+    #### Congratulations! You finished the first part of the tutorial, continue to learn more about Miden and understand how to send Peer-to-Peer private off-chain transactions!
+
+## Part 2: Peer-to-Peer private off-chain transactions
 
-Some of the tokens we minted can now be transferred to our second regular account. To do so, you can run:
+In this second part of the tutorial we will teach you how to make off-chain transactions and send funds to another account using the Miden client. 
+
+> **Note**:
+> We consider that you have done the first part of the turorial and that the state of your local client has not been reset. 
+
+
+### Setting-up the Miden client
+
+1. **Creating a second account:** To be able to send funds from one account to another we will need to generate one more account. For this example we will be generating 1 additional account: we generated `basic-immutable` account A in the first part of the tutorial. Here we will be creating `basic-immutable` account B using the following command:
+
+   ```shell
+   miden-client account new basic-immutable
+   ```
+
+   Please refer to the documentation of the CLI
+
+2. **Listing accounts:** To view the newly created accounts we can run the following command:
+
+   ```shell
+   miden-client account -l
+   ```
+
+   We should now see 2 available accounts listed:
+   - `basic-immutable` account A (created during the first part of this tutorial)
+   - `basic-immutable` account B
+
+### Transferring assets between accounts
+
+Now that we have two accounts we are ready to transfer some of the tokens we received from the faucet into account A. We will now transfer some to our second regular account B. To do so, you can run:
 
 ```shell
-miden-client sync # Make sure we have an updated view of the state
-miden-client tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 # Transfers 50 tokens to account ID B
+miden-client tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> <Amount>
 ```
 
-This will generate a Pay-to-ID (`P2ID`) note containing 50 assets, transferred from one regular account to the other. If we sync, we can now make use of the note and consume it for the receiving account:
+> **Note**
+> The faucet account id can be found on the faucet website
```

Comment:
How can I find the id?

Reply:
The website will be updated with the note id written under the title